src/types/utils/TextInputSelection.ts

```diff
@@ -0,0 +1,6 @@
+type TextInputSelection = {
```

Comment:
You can leave it in `src/libs/ComposerUtils/index.ts` and export it from there instead